A40-csds-support.md

```diff
@@ -3,8 +3,8 @@ xDS Configuration Dump via Client Status Discovery Service in gRPC
 * Author(s): lidizheng
 * Approver: markdroth
 * Status: In-Review
-* Implemented in: All languages
-* Last updated: 2021-02-19
+* Implemented in: Core/C++/Java/Golang (1st release)
```

Comment:
It is not actually implemented in any of these languages yet.  This field should be empty for now.

Reply:
Put to-be-done instead, delegated to the future self.